package.json

```diff
@@ -61,7 +61,7 @@
     "@endpass/connect": "^0.32.0-beta",
```

Comment:
![image](https://user-images.githubusercontent.com/13414205/83489141-a657da80-a4b6-11ea-868d-9c3b97b10dbb.png)

When I have locally uploaded Proof of address -- metrics were not sent